setup.cfg

```diff
@@ -47,6 +47,6 @@ edit_on_github = False
 github_project = astropy/specutils
 install_requires = astropy, six, gwcs, scipy
 # version should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
-version = 0.1.dev
+version = 0.3.1
```

Comment:
I believe the dev suffix is still needed though, isn't it?

Reply:
technically it's not, i don't think (``setup.py`` runs fine without it), but might be a process question about how versions get numbered.